docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -38,36 +38,37 @@ Be aware that if you are running out of time during your rental, you can [extend
 Before configuring color detection, enable the rover's camera to get a better sense of what it perceives.
 
 - If you are running this tutorial with a [rented Viam Rover](https://app.viam.com/try), enable both provided cameras: the front-facing camera and the overhead cam.
-  In the `viam_base` component panel under the **Control** tab, enable both the `cam` for the front-facing camera and the `overhead-cam:overheadcam` for an overhead view of your rover.
+  In the `viam_base` component panel under the **CONTROL** tab, enable both the `cam` for the front-facing camera and the `overhead-cam:overheadcam` for an overhead view of your rover.
 
   ![The viam_base component panel showing both the 'cam' and 'overheadcam' camera feeds enabled.](get-started/try-viam/try-viam/enable-both-cameras.png)
 
   You can also view and control the camera streams from the [individual camera component panels](/get-started/try-viam/try-viam-tutorial/#camera-control).
 
 - If you are running this tutorial on [your own Viam Rover](/get-started/try-viam/rover-resources/), enable the front facing camera.
-  If you are using the `ViamRover` [fragment](/get-started/try-viam/rover-resources/rover-tutorial-fragments/) with your rover, the front facing camera is named `cam` and can be enabled in the `viam_base` component panel under the **Control** tab.
+  If you are using the `ViamRover` [fragment](/get-started/try-viam/rover-resources/rover-tutorial-fragments/) with your rover, the front facing camera is named `cam` and can be enabled in the `viam_base` component panel under the **CONTROL** tab.
 
 ## Add the vision service to detect a color
 
+{{% alert title="Tip" color="tip" %}}
+If you are using a rented Viam rover, you can skip this section.
+Your rover is already configured with a `#7a4f5c` color detector called `my_color_detector`.
```

Comment:
Did you get a new rover? I just deployed a new one and did not have this in there. Could be that it's an a/b test thing but want to check that you didn't accidentally deployed a rover config you were previously testing on

Reply:
Ahhhh I think that's what happened, it was reused. I was confused because I don't remember ever working on the color detection tutorial before but I must have done so at some point with that rover. Will revert -- even if it was a/b that's still not every one. Thanks for testing!